include/boost/math/statistics/univariate_statistics.hpp

```diff
@@ -11,6 +11,7 @@
 #include <tuple>
 #include <cmath>
 #include <boost/assert.hpp>
+#include <boost/math/tools/precision.hpp>
```

Comment:
I'd probably just use `std::numeric_limits<Z>::max()` so that we don't have to bring in this header.